src/api/spec/models/user_spec.rb

```diff
@@ -278,10 +278,26 @@
         expect(user.email).to eq('some@email.com')
       end
     end
+  end
+
+  describe 'create_user_with_fake_pw' do
+    it_behaves_like 'successful login'
 
     context 'without params' do
+      it { expect(User.create_user_with_fake_pw.persisted?).to be false }
+    end
+  end
+
+  describe 'create_user_with_fake_pw!' do
+    it_behaves_like 'successful login'
+
+    context 'without params' do
+      before do
+        allow(User).to receive(:create_user_with_fake_pw).and_return(User.new)
```

Comment:
This is over-mocking. Instead would be better to pass invalid arguments to `create_user_with_fake_pw!` so that it raises an error.